### ISSUE TITLE
a11y: screen reader does not read aloud the copy icon when replacing a recovery key

### ISSUE THREAD

**Issue by matthew-hagemann** [canonical/desktop-security-center#127]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

Will the addition of https://github.com/canonical/desktop-security-center/pull/126, we have an issue where the screen ready is not correctly reading out the copy icon on the new recovery key when it has focus.

### Ubuntu release

None

### What architecture are you using?

None
